.github/workflows/run-python-script.yaml

```diff
@@ -0,0 +1,45 @@
+# This workflow will install Python dependencies and run the script
+
+name: Run script
+
+on:
+  workflow_dispatch: # add run button in github
+  schedule:
+    - cron: "32 10 * * *"
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.x"
+    - name: Install dependencies
```

Comment:
You might need to install GDAL separately like: https://github.com/OCHA-DAP/hdx-scraper-chirps/blob/68246b6ab3f1f52e2a23bdc71f80a24553bbe333/.github/workflows/run-python-script.yaml#L20
I also have had trouble with different python versions on GitHub Actions working with GDAL, so in my example I fixed python at 3.9.

Reply:
Yes good point I was thinking that, I've done it in the past but my way was a [bit more comples](https://github.com/OCHA-DAP/IBF_TYPHOON_DATA_PIPELINE/blob/main/.github/workflows/test-typhoon-model.yml#L40) so I'll give yours a shot if needed. I will wait to see if the pipeline complains, so far the tests seem to work at least.